server/re/src/reDataObjOpr.cpp

```diff
@@ -3269,19 +3269,16 @@ msiTarFileCreate( msParam_t *inpParam1, msParam_t *inpParam2, msParam_t *inpPara
 
 }
 
-/// Truncate a replica for the specified data object to the specified size.
-///
-/// \parblock
-/// This API selects a replica to truncate according to the rules of POSIX truncate(2). The caller may provide keywords
-/// via condInput in order to influence the hierarchy resolution for selecting a replica to truncate.
-/// \endparblock
+/// Truncate a replica for the specified data object to the specified size using #rs_replica_truncate.
```

Comment:
Mostly so that the reader has some other place to go for more information if desired. Don't have to link here, but I figured it wouldn't hurt. I defer.

Reply:
It's fine. Another benefit of including the API is it tells the reader which PEPs can be triggered.

Reply:
Okey dokey. I will leave it, then.

Reply:
Consider making that part of the body/details rather than the top line.

Reply:
Done